application/cms/form_fields.py

```diff
@@ -205,25 +221,21 @@ class RDUStringField(StringField):
     widget = _RDUTextInput()
 
     def __init__(
-        self,
-        label=None,
-        validators=None,
-        hint=None,
-        extended_hint=None,
-        strip_whitespace=False,
-        character_count_limit=None,
-        **kwargs,
+        self, label=None, validators=None, hint=None, extended_hint=None, character_count_limit=None, **kwargs
     ):
         # Automatically coalesce `None` values to blank strings
         # If we get null values from the database, we don't want to render these as 'None' strings in form fields.
         kwargs["filters"] = kwargs.get("filters", [])
         if _coerce_none_to_blank_string not in kwargs["filters"]:
             kwargs["filters"].append(_coerce_none_to_blank_string)
 
-        if strip_whitespace and _strip_whitespace not in kwargs["filters"]:
-            kwargs["filters"].append(_strip_whitespace)
-        elif not strip_whitespace and _strip_whitespace in kwargs["filters"]:
-            kwargs["filters"].remove(_strip_whitespace)
+        if self.__class__ is RDUStringField or self.__class__ is RDUURLField:
+            if _strip_whitespace_basic not in kwargs["filters"]:
```

Comment:
Just a wonder... why do we need to do the check if these are there already?  I don't think there's an obvious way this would already be set, and is there any harm in having it twice if it was already in kwargs?

Reply:
Yep, I think you're right. Tested it and it seems totally not necessary. 👍